src/components/IndexTable/README.md

```diff
@@ -200,6 +200,84 @@ function SimpleSmallScreenIndexTableExample() {
 }
 ```
 
+### Index table with row links
+
+A index table with rows that will link to a resource.
```

Comment:
Should we mention only a single `<Link />` component should be used in a given row?

```suggestion
An index table with rows that will link to a resource. The prop `primary` will allow the table to navigate without selecting the resource. Each row should contain a maximum of one primary link.
```